src/internal/storage/fileset/storage.go

```diff
@@ -479,3 +481,29 @@ func (s *Storage) Pin(tx *pachsql.Tx, fs ID) (PinnedFileset, error) {
 	}
 	return PinnedFileset(*id), nil
 }
+
+type ChunkSetID uint64
+
+func (s *Storage) CreateChunkSet(ctx context.Context, tx *sqlx.Tx) (ChunkSetID, error) {
+	ctx = pctx.Child(ctx, "createChunkset")
+	// Insert ChunkSet into ChunkSet table.
+	var chunksetID ChunkSetID
+	err := tx.GetContext(ctx, &chunksetID, `INSERT INTO storage.chunksets DEFAULT VALUES RETURNING id`)
```

Comment:
You can put the if before err as you do below.  This avoids `err` being bound in the rest of the function, which can sometimes cause confusion.